### DISCUSSION TITLE
Issues with indentation in PR view

### DISCUSSION THREAD

**Discussion by Naomi-Mathews** [community/community#52225] [Pull Requests]
I have 2 JS files both edited only by me in VS code editor. When i raise a pull request in github one js files shows tabs with 2 spaces and one displays tabs with 4 spaces. However if chose to edit or view file in github itself it shows the correct indentation

### File 1 working correctly
 Github PR view
<img width="1102" alt="Screenshot 2023-04-07 at 10 03 29 AM" src="https://user-images.githubusercontent.com/70324350/230541709-208b7eb9-80bd-4c3f-b03f-90d96babc170.png">
Editor
<img width="324" alt="Screenshot 2023-04-07 at 10 05 32 AM" src="https://user-images.githubusercontent.com/70324350/230541721-97b7eb98-5567-4075-8c7a-42fe070d50c3.png">

### File 2 behaving strangely in PR view
 Github PR view
<img width="1121" alt="Screenshot 2023-04-07 at 10 03 52 AM" src="https://user-images.githubusercontent.com/70324350/230541716-d75e3990-365d-455c-88fa-63a01cf9b8cd.png">
Github [view file] option
<img width="319" alt="Screenshot 2023-04-07 at 10 13 20 AM" src="https://user-images.githubusercontent.com/70324350/230542522-530a21f5-9594-4183-ae1f-c3c64d32a0a7.png">

Editor
<img width="313" alt="Screenshot 2023-04-07 at 10 04 46 AM" src="https://user-images.githubusercontent.com/70324350/230541719-1e4534fe-0c21-4a4e-8c0d-bc2a25f74fd0.png">

### Editor config
```
root = true

[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = tab
tab_width = 2

[*.js]
quote_type = single

[*.hbs]
quote_type = double
insert_final_newline = false

[*.{diff,md}]
trim_trailing_whitespace = false



```

**Comment by Spongman** [comment]
I see the same issue. here's a PR with a bunch of files, all indented with tabs. https://github.com/ThePrimeagen/ts-rust-zig-deez/pull/190/files

however, the PR diff view shows _some_ files correctly (2 spaces per tab, as specified in the .editorconfig) and other files _on the same page_ are rendered incorrectly (with 8 spaces).

for example, this is the first bad file: https://github.com/ThePrimeagen/ts-rust-zig-deez/pull/190/files#diff-780458dafdb5d3649b443fa69d1e562733eb41d272446bb7295c434815a980b3
![image](https://github.com/community/community/assets/1088194/64f5de3e-16eb-407e-9501-e5249f8c3925)

however clicking 'view file' shows it correctly:
![image](https://github.com/community/community/assets/1088194/f58f7c34-f938-40e4-b8fe-f7e7eab7ef15)
